src/integer_mod_q/polynomial_ring_zq/get.rs

```diff
@@ -99,3 +223,48 @@ mod test_get_value {
         assert_eq!(cmp_poly, poly_z);
     }
 }
+
+#[cfg(test)]
+mod test_get_degree {
+    use crate::{
+        integer::PolyOverZ,
+        integer_mod_q::{ModulusPolynomialRingZq, PolynomialRingZq},
+    };
+    use std::str::FromStr;
+
+    /// Ensure that degree is working.
+    #[test]
+    fn degree() {
+        let modulus = ModulusPolynomialRingZq::from_str("4  1 0 0 1 mod 17").unwrap();
+        let poly = PolyOverZ::from_str("4  -1 0 1 1").unwrap();
+        let poly_ring = PolynomialRingZq::from((&poly, &modulus));
+
+        let deg = poly_ring.get_degree();
+
+        assert_eq!(2, deg);
```

Comment:
Again, this that the degree is 2 is not obvious, change it so that it is easier to see.